lms/djangoapps/grades/signals/handlers.py

```diff
@@ -158,14 +159,13 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
     Handles the PROBLEM_WEIGHTED_SCORE_CHANGED signal by
     enqueueing a subsection update operation to occur asynchronously.
     """
-    result = recalculate_subsection_grade.apply_async(
+    result = _recalculate_subsection_grade.apply_async(
         kwargs=dict(
             user_id=kwargs['user_id'],
             course_id=kwargs['course_id'],
             usage_id=kwargs['usage_id'],
             only_if_higher=kwargs.get('only_if_higher'),
-            weighted_earned=kwargs.get('weighted_earned'),
-            weighted_possible=kwargs.get('weighted_possible'),
+            task_queued_time=unicode(datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")),
```

Comment:
Please create a helper for getting the current time as a string in util/date_utils.py.